Example/UINotifications-Example/ViewController.swift

```diff
@@ -78,13 +82,13 @@ final class ViewController: UIViewController {
     @IBOutlet private weak var titleTextField: UITextField!
     @IBOutlet private weak var subtitleTextField: UITextField!
     @IBOutlet private weak var showButton: UIButton!
-    
+
     @IBOutlet private weak var automaticallyDismissSwitch: UISwitch!
     @IBOutlet private weak var addActionSwitch: UISwitch!
     @IBOutlet private weak var addButtonSwitch: UISwitch!
     @IBOutlet private weak var showsImageSwitch: UISwitch!
     @IBOutlet private weak var successStyleSwitch: UISwitch!
-    
+
     override func viewDidLoad() {
         super.viewDidLoad()
         
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Example/UINotifications-Example/ViewController.swift;
  Line: 94;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)